### PR TITLE
Merge Image component

### PR DESCRIPTION
**Pull request by piyushsh25**

### CODE REVIEW

**Review by ZuberDunge — COMMENTED** [review]
Looks good.

File: components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+.img-rounded {
+    height: var(--size10R);
+    width: var(--size10R);
+    border-radius: var(--rounded);
+}
+
+.img-square {
+    height: var(--size10R);
+    width: var(--size10R);
+}
```
Comment (raghav2110) [inline components/image/image.css:17]: You can add extra line at the end as it is a good practice.Also red symbol goes away with that.

**Review by raghav2110 — COMMENTED** [review]
Overall code looks good.

File: components/image/image.css (hunk 1 above)
Comment (piyushsh25) [reply components/image/image.css:17]: The code formatter is not letting me create extra space at the end.

**Pull request merged**